### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by kevcenteno**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,113 @@
...
+ with either an invalid or missing IP Address.
+
+After creating the Transaction object, you can send a Score, Insights, or Factors
+request, which returns a Promise that contains the corresponding model:
+
+```
+// minFraud Score
+client.score(transaction).then(scoreResponse => ...);
+
+// minFraud Insights
```
Comment (andyjack) [inline README.md]:
```suggestion
// minFraud Insights
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,113 @@
...
+// minFraud Insights 
+client.insights(transaction).then(insightsResponse => ...);
+
+// minFraud Factors
+client.factors(transaction).then(factorsResponse => ...);
+
+```
+
+If the request fails, an error object will be returned in the catch in the form
+of:
```
Comment (andyjack) [inline README.md]: could you please strip this and other trailing spaces?

```suggestion
of:
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,113 @@
...
+
+## API Documentation
+
+Documentation for this API can be found [here](https://maxmind.github.io/minfraud-api-node/)
+
+## Usage
+To use this API, first create a new `Client` object. The constructor
+takes your MaxMind account ID and license key. For example:
+
+```
```
Comment (andyjack) [inline README.md]: Nice to have but not necessary, adding a language identifier to these code blocks.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,113 @@
...
+and Transaction property constructors.
 
-## Example
+In addition to the [response errors](https://dev.maxmind.com/minfraud/#Errors)
+returned by the web API, we also return: 
 
+```js
+{
+  code: 'SERVER_ERROR',
+  error: 'Some message',
```
Comment (andyjack) [inline README.md]: Oops, I missed this in the previous review, is `error` really "Some message" ?  Please correct if needed

Comment (kevcenteno) [reply README.md]: It's a stand-in for some dynamically generated message (which could change in the future).  Any suggestions to improve this?

Comment (wesrice) [reply README.md]: What about `<string>` or `...`?

**Pull request merged**